core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -256,6 +256,9 @@ trait EntityComponent {
         * @return
         */
       private def paginationSubquery(workspaceId: UUID, entityType: String, sortFieldName: String) = {
+        /* TODO: can we eliminate the SELECT ... e.all_attribute_values by being smarter about our where clauses?
+            while not returned to the end user, it still causes mysql extra work
+         */
```

Comment:
I'd like to tackle this in a separate, follow-on PR to avoid major scope-creep and risk of regression